db_upgrade_scripts/mosip_hotlist/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql

```diff
@@ -0,0 +1,97 @@
+CREATE DATABASE mosip_hotlist
+	ENCODING = 'UTF8'
+	LC_COLLATE = 'en_US.UTF-8'
+	LC_CTYPE = 'en_US.UTF-8'
+	TABLESPACE = pg_default
+	OWNER = postgres
+	TEMPLATE  = template0;
+
+COMMENT ON DATABASE mosip_hotlist IS 'Hotlist data related logs and the data is stored in this database';
+
+\c mosip_hotlist postgres
```

Comment:
Lets remove postgres in the above line, we are not giving the user as a new convention.

Reply:
Done